### PR TITLE
Add timeout to importer cmd wait for server ready

### PR DESCRIPTION
**Pull request by arnongilboa**
In case of timeout the importer container cmd exits with error and
termination message to be used by DV Running condition.

This is specifically for RegistryPullNode imports, used mostly in the
DataImportCron context.

PR #2044 was mistakenly reverted by PR #2141, so it's also fixed here.

Signed-off-by: Arnon Gilboa <agilboa@redhat.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: cmd/cdi-importer/importer.go [hunk 1]
```diff
@@ -111,8 +117,8 @@ func main() {
 			klog.Errorf("%+v", err)
 			os.Exit(1)
 		}
-
 	} else {
+		waitForReadyFile()
```
Comment (brybacki) [inline cmd/cdi-importer/importer.go:121]: good catch!

File: cmd/cdi-importer/importer.go [hunk 2]
```diff
@@ -41,16 +41,22 @@ func init() {
 }
 
 func waitForReadyFile() {
+	const readyFileTimeoutSeconds = 60
```
[the remark this answers is not in the document]
Comment (arnongilboa) [reply cmd/cdi-importer/importer.go:44]: We are waiting for `cdi-containerimage-server` http server readiness listening and serving the container image itself to be imported by `cdi-importer` in the same pod. If for example k8s cannot pull the server container image from its source, the server will never run, so its readyFile will never be created and the importer will be stuck forever. Using this timeout, the importer exits with error and termination message which is already passed to the DV Running condition. The container is restarted by k8s but CDI can decide whether to delete this DV based on the running condition.

**Pull request merged**